source/getting-started/admin-onboarding-tasks.rst

```diff
@@ -13,146 +13,202 @@ Administrator Onboarding Tasks
+You already have your users grouped by role, location, or level, so Mattermost provides identity management, single sign-on, and automatic account provisioning to make it easy for you to integrate with your existing identity and access management (IAM) services and systems with `Active Directory and LDAP <https://docs.mattermost.com/onboard/ad-ldap.html>`__ and `SAML 2.0 SSO <https://docs.mattermost.com/onboard/sso-saml.html>`__ integrations featuring providers like `Active Directory Federation Services <https://docs.mattermost.com/onboard/ad-ldap.html#configure-ad-ldap-deployments-with-multiple-domains>`__, `Okta <https://docs.mattermost.com/onboard/sso-saml-okta.html>`__, `GitLab <https://docs.mattermost.com/onboard/sso-gitlab.html>`__, `Google <https://docs.mattermost.com/onboard/sso-google.html>`__, and `Office 365 <https://docs.mattermost.com/onboard/sso-office.html>`__
 
-**Checking for a valid license in Enterprise Edition without logging in**
+- Begin to onboard users by `enabling account creation <https://docs.mattermost.com/configure/configuration-settings.html#enable-account-creation>`__ or by connecting an authentication service to assist with user provisioning.
 
-- Open the log file ``mattermost.log``. It's usually in the ``mattermost/logs/`` directory but might be elsewhere on your system. Find the last occurrence of a log entry that starts with the text ``[INFO] License key``. If the license key is valid, the complete line should be similar to the following example:
+- For bulk onboarding, enable `AD/LDAP Group Synchronization <https://docs.mattermost.com/configure/configuration-settings.html#enable-ad-ldap-group-sync>`__ to ensure new users are added to default teams and channels as they join Mattermost. See our `AD/LDAP Groups <https://docs.mattermost.com/onboard/ad-ldap-groups-synchronization.html>`__ documentation to learn more.
 
-.. code-block:: text
+- See our `migration guide <https://docs.mattermost.com/onboard/migrating-to-mattermost.html#migration-guide>`_ and `bulk loading documentation <https://docs.mattermost.com/onboard/bulk-loading-data.html>`_ for additional details.
 
-  [2017/05/19 16:51:40 UTC] [INFO] License key valid unlocking enterprise features.
-      
-User Experience Optimizations
------------------------------
+Control product access with permissions
+----------------------------------------
 
-We highly recommend the following best practices, configuration options, and features for an optimal Mattermost user experience.
+You can customize which users can perform specific actions, such as creating teams, managing channels, posting reactions, `adding custom emojis <https://docs.mattermost.com/configure/configuration-settings.html#enable-custom-emoji>`__, or editing or deleting posts. See our `Advanced Permissions <https://docs.mattermost.com/onboard/advanced-permissions.html>`__ documentation for details.
 
-**1. Upgrade your Mattermost server**
+Team creation
```

Comment:
Is the intent for this section to recommend creating a public and private team? There isn't much detail in the team creation page about the use case for a public and private team.